zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/AQLGenerator.scala

```diff
@@ -121,9 +135,9 @@ class AQLGenerator extends IQLGenerator {
         """.stripMargin
       )
       //TODO check if the vars are duplicated
-      val field: Field = ??? // get field from lookup table
+      val fieldMap: Map[String, Field] = schemaMap.flatMap(_._2.fieldMap) // get fields from lookup and query table
```

Comment:
@JavierJia How will the parser know which schema the select name is referring to?
eg. Emp(id, name, salary, dept_id) & Dept(id, name)
the lookup JSON will be
{
          "joinKey":"dept_id",
          "dataset":"Dept",
          "lookupKey":"id",
          "select":"name",
          "as" : "name"
}

Which name should we return here?